Project.toml

```diff
@@ -13,10 +13,10 @@ RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 
 [compat]
 AbstractOperators = "0.1 - 0.2"
```

Comment:
```suggestion
AbstractOperators = "0.3"
```

Reply:
Sure. In fact, I requested a version bump on AbstractOperators, I'm glad it went through so quickly.